docs/iris/src/index.rst

```diff
@@ -44,11 +38,61 @@ Interoperability with packages from the wider scientific Python ecosystem comes
 from Iris' use of standard NumPy/dask arrays as its underlying data storage.
 
 Iris is part of SciTools, for more information see https://scitools.org.uk/.
+For **Iris 2.4** and earlier documentation please see the
+:link-badge:`https://scitools.org.uk/iris/docs/v2.4.0/,"legacy documentation",cls=badge-info text-white`.
+
+
+.. panels::
+    :container: container-lg pb-3
+    :column: col-lg-4 col-md-4 col-sm-6 col-xs-12 p-2
+
+    Install Iris to use or for development.
+    +++
+    .. link-button:: installing_iris
+        :type: ref
+        :text: Installing Iris
+        :classes: btn-outline-primary btn-block
+    ---
+    View the gallery that includes python code used to create it.
+    +++
+    .. link-button:: sphx_glr_generated_gallery
+        :type: ref
+        :text: Gallery
+        :classes: btn-outline-primary btn-block
+    ---
+    Find out what has recently been added to Iris, or soon will be.
+    +++
+    .. link-button:: iris_whatsnew
+        :type: ref
+        :text: What's new
+        :classes: btn-outline-primary btn-block
+    ---
+    Learn how to use Iris.
+    +++
+    .. link-button:: user_guide_index
+        :type: ref
+        :text: User Guide
+        :classes: btn-outline-primary btn-block
+    ---
+    Documentation for the Iris Application Programming Interface.
```

Comment:
I'm not sure whether API is a familiar term for a lot of our target user-base.  Maybe something like "Browse full Iris functionality by module"?